src/components/UserStatistics.js

```diff
@@ -105,6 +105,9 @@ function UserStatistics({ gamesData, userId }) {
           Average game length:{" "}
           <strong>{stats[0] ? formatTime(stats[4] / stats[0]) : "N/A"}</strong>
         </Typography>
+        <Typography varient="body2" className={classes.statsItem}>
```

Comment:
Typo here. `varient` should be `variant`.

Reply:
Good catch, I'll change it on Monday (TODO).